### PR TITLE
Ability to set event timezone #2512

### PR DESCRIPTION
**Pull request by leonardohof**
Resolve opencollective/opencollective#2512

### CODE REVIEW

**Comment by leonardohof** [comment]
@znarf Could you review, please?

**Comment by znarf** [comment]
Looks good. Can you share a screenshot?

**Comment by leonardohof** [comment]
image

image

**Comment by Betree** [comment]
Tested, the feature works. Thanks @leonardohof :slightly_smiling_face:

File: components/TimezonePicker.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormGroup, ControlLabel, FormControl } from 'react-bootstrap';
```
[the remark this answers is not in the document]
Comment (leonardohof) [reply components/TimezonePicker.js]: @Betree Okay.

[the remark this answers is not in the document]
Comment (leonardohof) [reply components/TimezonePicker.js]: @Betree I changed to use the StyledSelect. Take a look.

**Comment by leonardohof** [comment]
@znarf @Betree any updates?

**Pull request merged**